openapi/openapi.yaml

```diff
@@ -80,6 +80,11 @@ tags:
     description: |
       Create and manage school news (also known as communications).
 
+  - name: notification
```

Comment:
We probably don't need a new section, the endpoint exists under the 'user' area so we can probably just keep this as a part of the 'User' section.